multigeiger/multigeiger.ino

```diff
@@ -211,7 +217,12 @@ void loop() {
     last_GMC_counts = GMC_counts;
   }
 
-  iotWebConf.doLoop();  // see webconf.cpp
+  long loop_duration;
+  loop_duration = millis() - current_ms;
+  iotWebConf.delay((loop_duration < LOOP_DURATION) ? (LOOP_DURATION - loop_duration) : 0);
+  loop_duration = millis() - current_ms;
+  log(DEBUG, "loop_duration: %ld", loop_duration);
```

Comment:
Would set it to INFO

Reply:
this is nothing interesting for a normal user, thus debug is better.

Reply:
then remove it: you can't hardly see any other debug message, if this comes every 5msec

Reply:
ok, i'll remove output and computation.

doing an INFO-level log() call would not have helped to not see it, because INFO level stuff is of course also seen when the logger is on DEBUG log level.

Reply:
OK, you're right. I always think the wrong way with debug levels :-(